server/src/main/java/org/elasticsearch/action/bulk/TransportBulkAction.java

```diff
@@ -579,7 +589,7 @@ void createIndex(String index, boolean requireDataStream, TimeValue timeout, Act
     void rolloverDataStream(String dataStream, TimeValue timeout, ActionListener<RolloverResponse> listener) {
         RolloverRequest rolloverRequest = new RolloverRequest(dataStream, null);
         rolloverRequest.masterNodeTimeout(timeout);
-        client.execute(RolloverAction.INSTANCE, rolloverRequest, listener);
+        client.execute(AutoRolloverAction.INSTANCE, rolloverRequest, listener);
```

Comment:
Can you add a comment above this line about why this uses `AutoRolloverAction` instead of the regular `RolloverAction`? (I want to avoid it getting accidentally undone or misunderstood in the future)